core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -260,6 +229,12 @@ class RequestChannel(val numProcessors: Int, val queueSize: Int) extends KafkaMe
 
   /** Send a response back to the socket server to be sent over the network */
   def sendResponse(response: RequestChannel.Response) {
+    if (isTraceEnabled) {
+      val requestHeader = response.request.header
+      val apiKey = ApiKeys.forId(requestHeader.apiKey)
+      trace(s"Sending $apiKey to client ${requestHeader.clientId} of ${response.responseSend.size} bytes.")
```

Comment:
Would this read a bit better if we said s"Sending $apiKey response to client..."